pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -114,6 +114,24 @@ def _is_inside_context_manager(node):
     )
 
 
+def _will_be_released_automatically(node: astroid.Call) -> bool:
+    """Checks if a call that could be used in a ``with`` statement is used in an alternative
+    construct which would ensure that its __exit__ method is called."""
+    callables_taking_care_of_exit = frozenset(
+        (
+            "contextlib._BaseExitStack.enter_context",
+            "contextlib.ExitStack.enter_context",  # necessary for Python 3.6 compatibility
+        )
+    )
+    parent = node.parent
```

Comment:
As it's just used two times, it's probably better to write out `node.parent` directly. That way you don't have to think about it.